tests/modules.rs

```diff
@@ -4,16 +4,24 @@ use frame_support::assert_ok;
 use mock::*;
 use move_core_types::{identifier::Identifier, language_storage::StructTag};
 use pallet_move::address;
+use sp_runtime::AccountId32;
 
-const EMPTY_ADDR: u64 = 0x000000000CAFE_u64.to_be();
+const EMPTY_ADDR: AccountId32 = AccountId32::new([1u8; 32]);
 
```

Comment:
So, the address here must match the address in the `tests/assets/move/Move.toml` for the published module.
That's the reason why publishing fails right now.

We could keep the original const`0xCAFE` here (the Move address in TOML) and then before we call the publish API in our failing tests, we first convert the original const to the AccountID with new conversion functions, and then the tests should pass.

Or we start using some new SS58 address in tests and then update the const here.

Reply:
Updated to use [1u8; 32] fixture